src/app/dataset-view/dataset.component.ts

```diff
@@ -100,11 +50,10 @@ export class DatasetComponent implements OnInit, OnDestroy {
     public isAvailableLinageGraph = false;
     public headings: Element[] | undefined;
     public isMarkdownEditView = false;
-    public clipboardKamuCli = "kamu pull kamu.dev/anonymous/dataset";
-    public clipboardKafka = "https://api.kamu.dev/kafka/anonymous/dataset";
     public markdownText = AppValues.markdownContain;
 
     private w: Window;
+    private subs: SubSink = new SubSink();
```

Comment:
Not all subscriptions in this component became part of the `subs`

Reply:
fixed